assets/js/components/surveys/CurrentSurvey.js

```diff
@@ -264,9 +266,11 @@ export default function CurrentSurvey() {
 				<SurveyQuestionComponent
 					key={ currentQuestion.question_text }
 					answerQuestion={ answerQuestion }
-					choices={ currentQuestion.question.answer_choice }
+					choices={ currentQuestion.question?.answer_choice } // eslint-disable-line camelcase
 					dismissSurvey={ dismissSurvey }
 					question={ currentQuestion.question_text }
+					subtitle={ currentQuestion.subtitle }
+					placeholder={ currentQuestion.placeholder }
```

Comment:
This part wasn't mentioned in the IB. Passing in every prop required for every type of QuestionComponent doesn't seem the ideal approach (especially when each question won't have these values)

There are better ways to do this *but* I don't want to generate merge conflicts, so I just did the minimum for the tests to pass :+1:

Reply:
I'm not a fan of prop spreading, but I suppose assembling an object for each question type then spreading that isn't the end of the world.

I'd be fine with that if you wanted to, but I don't really mind the explicitness of passing all of the props 😅

Reply:
Currently, there are three open PRs all with similar changes here.

IMHO we should just have a switch statement that returns the actual component instantiated with the correct props, rather than this `SurveyQuestionComponent` trickery :smile: 

But I presume this will be fixed when 

I only use prop spreading to allow setting of HTML attributes (when creating a text input or something like that) :+1: